features/admin.tenants.v1/api/tenants.ts

```diff
@@ -160,3 +161,25 @@ export const getAssociatedTenants = (
             return Promise.reject(error?.response?.data);
         });
 };
+
+/**
+ * Get the deployment units.
+ *
+ * @returns - A promise that resolves with the Deployment response object.
+ */
+export const getDeploymentUnits = (
```

Comment:
we use SWR for http get requests. Therefore, shall we refactor this method to a separate hook that utilize useRequest hook?